README.md

```diff
@@ -0,0 +1,210 @@
+fine-grained data access down to the level of individual files, and allows for  
+tracking future updates. In order to use this repository for data retrieval,  
+[DataLad](https://www.datalad.org/) is required. It is a free and open source  
+command line tool, available for all major operating systems, and builds up on  
+Git and [git-annex](https://git-annex.branchable.com/) to allow sharing,  
+synchronizing, and version controlling collections of large files.  
+
+## Installation
+
+- Install datalad according to the [DataLad handbook](https://handbook.datalad.org/en/latest/intro/installation.html). It is recommended to install globally. 
+- Install [Python](https://www.python.org)
+- [pydoit](https://pydoit.org/install.html)
+
+Note that for [simply downloading the dataset,](#1-easy-access) Python and pydoit are not required.
+
+## Getting Started
+
+### Clone the repository
+
+A DataLad dataset can be `cloned` by running  
+```
+datalad clone
+```
+Do not use **git clone** to download the repository! This way DataLad will not have the necessary
+information to run the program. Once a dataset is cloned, it is a light-weight directory on your local machine.  
+At this point, it contains only small metadata and information on the identity  
+of the files in the dataset, but not actual *content* of the (sometimes large)  
+data files.  
+
+
+<h3 id="1-easy-access"> Easy access </h3>
+Users who simply want to retrieve the dataset have the option to access both the
+original and extracted files with
+
+```
+dataland get <filename>
+```
+This command will trigger a download of the files, directories, or subdatasets  
+you have specified.
+
+For example, the CSV file for the 2023/09/13 release can be downloaded with
+```
+datalad get extracted_data/v230913/Robbie_Andrew_Cement_Production_CO2_230913.csv 
+```
+### Stay up-to-date  
+  
+DataLad datasets can be updated. The command `datalad update` will *fetch*  
+updates and store them on a different branch (by default  
+`remotes/origin/master`). Running  
+  
+```  
+datalad update --merge  
+```  
+  
+will *pull* available updates and integrate them in one go.  
+  
+### Find out what has been done  
+  
+DataLad datasets contain their history in the ``git log``.  By running ``git  
+log`` (or a tool that displays Git history) in the dataset or on specific  
+files, you can find out what has been done to the dataset or to individual  
+files by whom, and when.
+
+<h2 id="2-contributing"> Contributing </h2>
+
+For those who wish to contribute to the repository, below we go through the key commands you will need to use. 
+
+#### Set up the virtual environment with doit
+```
+doit setup_env
+```
+#### <a name="download"></a>Download the version from the command line.
+This will download all files from Zenodo as they are for a specific version (note this version must already be in `versions.py`, if you want to add a new version, see the section on adding a new version below).
```

Comment:
```suggestion
This will download all files from Zenodo as they are for a specific version (note this version must already be in `versions.py`, if you want to add a new version, see the section on adding a new version below).

```